### ISSUE TITLE
 In Step6 GitHub Issues Tutorial - to delete the defunct branch in your local repository.

### ISSUE THREAD

**Issue by apurva-g** [open-learning-exchange/open-learning-exchange.github.io#1021]
After the pull request was merged I deleted the defunct branch in my remote repository and when I tried to delete the defunct branch in my local repository using this command
**git branch -d yourLocalBranchName**, it gave an error saying "cannot delete the branch which you are currently on". So I was trying to know how to delete that branch on my local repository, I feel we should have those steps explained in the tutorial which would help interns to overcome that error.

**Comment by apurva-g** [comment]
I followed these steps to delete the branch on my local repository and was able to delete. I think adding this screenshot in the tutorial would be helpful.

![screen shot 2017-07-01 at 3 18 42 pm](https://user-images.githubusercontent.com/22311158/27764801-bbf5d3f4-5e70-11e7-8210-5166fb5827f6.png)

**Comment by tjyothsna16** [comment]
These steps are helpful, but personally, I think it's better to write down the advantages and disadvantages on the same page. 
For example, I can see a disadvantage here, if we delete any branch we can not use the deleted branch rawgit link anymore. 
One of the main advantages I can see here is, we can just use the name of the deleted branch to create a new branch. 
So, feel free to list the advantages and disadvantages on the same page as it will help people to decide whether to delete or not. 
You can also create a PR as well.

**Comment by apurva-g** [comment]
Sure, that's a good point. Thank you. I will add that as well and create a pull request. Thanks.

**Comment by kylemathias** [comment]
What part in the first steps are you referring to in deleting a branch? I don't know if its necessary to enplane how to delete a local branch when deleting it on github is what is more important. Changing branches and what errors you might get could create an endless list of things to add to the first steps. I would not make this change.

**Comment by apurva-g** [comment]
Hi, Kyle! I am talking about deleting a local branch that we would have created to work on an issue. 
Once the issue is resolved and its pull request is merged we no longer need that branch. And yes as you said we would delete that branch from Github but those are the steps explained to delete it locally. I thought of adding it because in the tutorial already there is a command given which tells how to delete the branch locally but that was giving me an error. After I tried those above steps in the screenshot it worked. Do you think I should give a link to that page rather than explaining it if it is too much content?

**Comment by kylemathias** [comment]
I think a link would be more appropriate if it is becoming necessary to explain how to delete a local branch. We want people to use the online github branching and deleting functions there. if someone wants to us the local branch system, they should be looking it up themselves.

**Comment by apurva-g** [comment]
Ok, I will add a link and create a pull request.

**Comment by sarah-lu102** [comment]
I agree with @kylemathias , I think the steps already have so much information that it is better to just let interns google the solution to deleting branches- being able to figure things out on their own is also important.

**Comment by apurva-g** [comment]
So do you all think I should just provide a link to it on the page and create a pull request in order to complete the first steps or I need to create another issue? Either is fine with me.

**Comment by kylemathias** [comment]
I would say make a link at the most. Something like "Go here to find out more about how to delete a branch locally"

**Comment by apurva-g** [comment]
Sure. Thank you.

**Comment by dogi** [comment]
@apurva-g was wondering where we stand on this issue?

**Comment by alexkarasik** [comment]
@apurva-g Are you still working on this? I will close it by Wednesday if you are not working on it.
